build/state_data_types_partial.md

```diff
@@ -0,0 +1,75 @@
+## Data types
```

Comment:
Does it also support expressions?

Reply:
Not as the part of this proposal. 

For now I `global-state` to work similarly to `get`, meaning returned value will be interpreted as JSON types:
>  The literal operator is not necessary if the array is returned from a sub-expression, e.g. ["in", 1, ["get", "myArrayProp"]].

It seems feasible to support expressions, but I did not investigate that.